scripts/linux/trcE2ETest.sh

```diff
@@ -300,74 +303,72 @@ function print_test_run_logs() {
         return;
     fi
 
-    print_highlighted_message 'LOGS FROM IOTEDGED'
-    journalctl -u iotedge -u docker --since "$test_start_time" --no-pager || true
-
-    print_highlighted_message 'edgeAgent LOGS'
-    docker logs edgeAgent || true
+    print_deployment_logs
```

Comment:
This is the key change -- call print_deployment_logs from print_test_run_logs() so that logs from all iotedge system services will be printed. Prior to this change, only logs form docker and deployed edge modules were printed when connectivity tests failed.